doc/user-guide.md

```diff
@@ -63,6 +63,17 @@ By default this will be the namespace that the operator is running in. To watch
 mgr, err := manager.New(cfg, manager.Options{Namespace: ""})
 ```
 
+It is also possible to use the [MultiNamespacedCacheBuilder](https://godoc.org/github.com/kubernetes-sigs/controller-runtime/pkg/cache#MultiNamespacedCacheBuilder) to watch a specific set of namespaces:
```

Comment:
We're using [reference style links](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links) throughout the rest of the user-guide to make the raw markdown easier to read.
So perhaps change this to a reference link.

Reply:
done